### PR TITLE
Post data

### PR DESCRIPTION
**Pull request by percworld**

### CODE REVIEW

File: src/APICalls.js [hunk 1]
```diff
@@ -39,4 +39,9 @@ const fetchData = () => {
         });
 }
 
-export default fetchData;
+export const displayError = (errorMessage) => {
+  const updatingDisplay =  document.getElementById('updatingDisplay');
+  const message = errorMessage.message === 'Failed to fetch' ?
+    "Internet connection may be unstable. Check again in a moment please." : errorMessage
+  updatingDisplay.innerText = message;
+}
```
Comment (PaigeVannelli) [inline src/APICalls.js:47]: This looks great!

File: src/scripts.js [hunk 2]
```diff
@@ -130,39 +129,46 @@ function postNewData() {
...
-    updatingDisplay.innerHTML = "Updating Your Account...";
-    setTimeout(() => {updatingDisplay.innerHTML = ""}, 1500)
-    console.log(response)
-  })
-  .catch(err => {
-    alert(err.message)
-  })
+  if (!newEntry.hoursSlept || !newEntry.sleepQuality || !newEntry.numOunces || !newEntry.numSteps || !newEntry.minutesActive || !newEntry.flightsOfStairs) {
+    updatingDisplay.innerText = "Please make sure fields are all filled."
+  } else {
```
Comment (PaigeVannelli) [inline src/scripts.js:162]: For future iterations we could do error handling so that user can't put in anything but numbers

**Review by PaigeVannelli — APPROVED** [review]
Looks good! I think this is a good balance of error handling multiple potential errors

**Comment by josharagon** [comment]
Work. displays data as intended. Great job.

**Pull request merged**